### DISCUSSION TITLE
安可「アンコール」　Update

### DISCUSSION THREAD

**Discussion by mirai-mamori** [mirai-mamori/Sakurairo#196] [General]
全新的无 Jquery 主题体验
全新的Webpack化主题体验
微定制设置框架，降低设置搜索阈值，优化性能。
全新的独立相册模版（体验版）
增强的代码高亮功能
增强的深色模式体验

新增
- 文章页面自动添加菜单功能
- 文章区域多作者信息
- 导航菜单毛玻璃效果
- 公开测试频道
- 前台音乐播放器相关功能
- 验证码验证（替代滑动验证）

优化
- 全局界面动画
- PHP8的兼容性
- 古腾堡相关CSS
- 主题文件结构
- 部分CSS引入全局变量
- 默认设置值及相关说明

修正
- issue模版中的文本错误
- 导航菜单登录菜单♻️过快的特性
- 文章没有Large尺寸的缩略图时的报错情况
- 评论头像概率炸图的问题
- 手机端导航菜单的一些bug

<hr /><em>This discussion was created from the release <a href='https://github.com/mirai-mamori/Sakurairo/releases/tag/2.1.0'>安可「アンコール」　Update</a>.</em>

**Comment by xiaoman1221** [comment]
来了

**Comment by shaokeyibb** [comment]
好耶
